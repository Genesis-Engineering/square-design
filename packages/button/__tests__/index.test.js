import React from 'react';
import { render } from '@testing-library/react';
import Button from '../lib';
import '@testing-library/jest-dom';

describe('Button Component', function () {
    let buttonContainer;

    beforeEach(() => {
        const { container } = render(<Button>this is</Button>);
        buttonContainer = container;
    });

    it('it should render a component', async () => {
        expect(buttonContainer).toBeInTheDocument();
        expect(buttonContainer).toMatchSnapshot();
    });
});
