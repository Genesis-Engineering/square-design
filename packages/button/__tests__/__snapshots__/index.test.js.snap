// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component it should render a component 1`] = `
<div>
  <button
    class="sc-bczRLJ jMiuxo ge-button ge-button__primary  "
  >
    this is
  </button>
</div>
`;
